### PR TITLE
Add exported panel with yarprobotinterface logging table

### PR DESCRIPTION
**Pull request by nunoguedelha**
This implementation requires minor changes and illustrates well how to add pre-exported panels in the domain objects left pane of the Yarp-OpenMCT visualiser.

## Summary of changes

1. Add the exported `.json` file to the folder `<repo-root-path>/exchange/<arbitrary-created-sub-folder>`,
2. Add respective sub-folder and telemetry entry in the dictionary `dictionaryMySavedPanels.json`
   - dictionary root object name/id: "My Saved Panels"/"mySavedPanels"
   - sub-folder object name/id: "Logging Tables"/"loggingTables"
   - telemetry entry (leaf) namespace:key = "myPanel300:root", which has always to be unique
   If a new dictionary had been created, it would have to be added to the map file https://github.com/ami-iit/yarp-openmct/blob/fbd69b24de04b631ad86016161c69d6414c830fd/openmctStaticServer/plugins/namespacesNtypes.js#L4-L10
4. Add an instance of the plugin `StaticImportObjectPlugin` providing, as function inputs:
    - the namespace such that `<namespace>:root` is laways unique,
    - the exported file path relative to the repo rooth path,
    - the root location `namespace:id` of the imported metadata: `yarpopenmct:mySavedPanels`.

### CODE REVIEW

File: openmctStaticServer/plugins/conf/dictionaryMySavedPanels.json [hunk 1]
```diff
@@ -35,6 +35,14 @@
       "telemetryEntries": [
         {"namespace": "myPanel200", "key": "root"}
       ]
+    },
+    {
+      "name": "Logging Tables",
```
Comment (S-Dafarra) [inline openmctStaticServer/plugins/conf/dictionaryMySavedPanels.json:40]: Just a small comment. The name is a bit generic

Comment (nunoguedelha) [reply openmctStaticServer/plugins/conf/dictionaryMySavedPanels.json:40]: Indeed, "Process Yarp Text Logging Tables" maybe?

Comment (S-Dafarra) [reply openmctStaticServer/plugins/conf/dictionaryMySavedPanels.json:40]: Sorry, I misunderstood. That's the name of the folder. That's perfectly fine as it is

**Review by S-Dafarra — APPROVED** [review]

**Comment by nunoguedelha** [comment]
Thanks @S-Dafarra !

**Pull request merged**